### PR TITLE
 International Expansion Phase 3 Group 2 - Add support for 3 new countries in WCPay

### PR DESCRIPTION
**Pull request by elazzabi**
#### Changes proposed in this Pull Request

Add support for:

- Bulgaria (BG)
- Croatia (HR)
- Romania (RO)

#### Testing instructions

* No need for testing. Each country was tested individually.

#### Remaining tasks

None

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-5.9.0#adding-support-for-3-new-countries-in-wc-payments
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6215` or branch name `add/intl-expansion-phase3-group2` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=add/intl-expansion-phase3-group2) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: ea94afa01b72379d8a6632ff5d80414cd2e13ced
- Build time: 2023-05-11 07:37:13 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: client/utils/account-fees.tsx [hunk 1]
```diff
@@ -25,8 +25,10 @@ const countryFeeStripeDocsSectionNumbers: Record< string, string > = {
 	AU: 'australia',
 	AT: 'austria',
 	BE: 'belgium',
+	BG: 'bulgaria',
 	CA: 'canada',
 	CY: 'cyprus',
+	CZ: 'czechia',
```
Comment (anu-rock) [inline client/utils/account-fees.tsx]: Can we double check if it's czechia or czech-republic?

**Review by anu-rock — APPROVED** [review]
LGTM 🎉

**Review by dpaun1985 — APPROVED** [review]
✅ LGTM

**Review by mordeth — APPROVED** [review]
LGTM! ✅

File: client/utils/account-fees.tsx (hunk 1 above)
Comment (elazzabi) [reply client/utils/account-fees.tsx]: Double-checked with @aheckler, this should be `czech-republic`. Fixed in https://github.com/Automattic/woocommerce-payments/pull/6215/commits/e4fe789d64faf59a175f75de6dff1ffe424b4771

Comment (aheckler) [reply client/utils/account-fees.tsx]: Yeah, core WC still uses `Czech Republic` ([link](https://github.com/woocommerce/woocommerce/blob/7.6.1/plugins/woocommerce/i18n/countries.php#L76)) so let's stick with that for now. If core changes, so can WC Pay.

**Comment by elazzabi** [comment]
Added an error notice if the store is using any currency with decimal problems: CZK, HUF, and SEK

https://github.com/Automattic/woocommerce-payments/pull/6215/commits/b1b4c41101b15d86723db458111e2adb7856791d

**Comment by elazzabi** [comment]
Removed support for Sweden, Czech Republic, and Hungary. Updated changelog and PR title and description

**Comment by elazzabi** [comment]
Testing instructions https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-5.9.0#adding-support-for-3-new-countries-in-wc-payments

**Pull request merged**